docs/core/integration-quality-scale/rules/iqs046.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IQS046 - The documentation provides a list of known supported / unsupported devices"
+---
+
+IQS046 is as follows:
+
+The documentation provides a list of known supported / unsupported devices
+
+## Reasoning
+
+A lot of Home Assistant users buy devices based on if Home Assistant supports them.
+To make it easier for users to find out if a device is supported, the documentation should provide a list of known supported or unsupported devices.
+This would decrease the amount of bad experiences where the user finds out their device is not supported when they try to set it up.
```

Comment:
```suggestion
This will decrease the amount of bad experiences where the user finds out their device is not supported when they try to set it up.
```